frontend/src/views/WheresMyGeneV2/components/InfoPanel/components/SourceData/style.ts

```diff
@@ -38,6 +38,12 @@ export const StyledLink = styled.a`
   font-weight: 400;
 `;
 
+export const StyledSpan = styled.span`
+  font-size: 14px;
+  font-weight: 400;
+  color: ${() => warning500};
```

Comment:
The following will do!

```suggestion
  color: ${warning500};
```